test/public/envs/overview.test.js

```diff
@@ -130,24 +134,22 @@ module.exports = () => {
     });
 
     it('can set how many environments are available per page', async () => {
-        await waitForTimeout(300);
         // Expect the amount selector to currently be set to 10 (because of the defined page height)
         const amountSelectorId = '#amountSelector';
         const amountSelectorButton = await page.waitForSelector(`${amountSelectorId} button`);
         const amountSelectorButtonText = await page.evaluate((element) => element.innerText, amountSelectorButton);
-        await waitForTimeout(300);
         expect(amountSelectorButtonText.trim().endsWith('10')).to.be.true;
 
         // Expect the dropdown options to be visible when it is selected
         await amountSelectorButton.evaluate((button) => button.click());
-        await waitForTimeout(100);
+        await page.waitForSelector(`${amountSelectorId} .dropup-menu`);
         const amountSelectorDropdown = await page.$(`${amountSelectorId} .dropup-menu`);
         expect(Boolean(amountSelectorDropdown)).to.be.true;
```

Comment:
This construction should simply be replaced by:
```await page.waitForSelector(`${amountSelectorId} .dropup-menu`);```

Reply:
Done